bash/bldjar

```diff
@@ -1,53 +1,38 @@
 #!/bin/bash
 
 # bldjar: rebuild z390.jar from src using current installed JDK
-#
-# Note: must be run from z390/bash subdirectory
 
-read -s -p "Press Enter to reuild z390.jar from src using current installed JDK";
-echo "";
+set -e              # exit on all errors
+cd $(dirname $0)    # set to directory of script
 
-# Windows BLDJAR.BAT copied src *.java and Z390.MAN to jar; not done here
+mkdir -p "../jar"   # create folder if does not exist
 
-# ensure destination directory exists; exit if not
-if [ ! -d "../jar" ]; then
-    echo "Error: ../jar directory does not exist"
-    exit 1
-fi
 # empty destination directory; cd to it
-rm ../jar/*
+rm -f ../jar/*
 cd ../jar
+
 # compile z390 java sources to class vm object code
 echo "Compiling ../src/*.java into jar"
 javac -d . -g:none ../src/*.java
 ec=$?
 echo "The exit code of the javac command is $ec"
-# pause to allow the user to terminate
-read -s -p "Press Enter to continue; press Ctrl-C to exit"
-echo ""
-echo ""
+
+# create z390.properties file
+../bash/relver
```

Comment:
It is my opinion that you cannot do this. See comments for relver.

Reply:
Alternatives requested.